src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/CategoryCard/components/InstanceTypesMenu/CustomMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ElementType, useRef, useState } from 'react';
+
+import CustomMenuRenderer from '@catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/CategoryCard/components/InstanceTypesMenu/CustomMenuRenderer';
+import EnhancedMenuToggle from '@catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/CategoryCard/components/InstanceTypesMenu/EnhancedMenuToggle';
```

Comment:
isnt the imports just ../ ? instead of using the alias?

Reply:
My IDE creates the imports automatically and I didn't pay attention to them. I can go in and make them relative.

Reply:
Fixed!